test/vue-test-util.plugin.ts

```diff
@@ -0,0 +1,43 @@
+import { config } from '@vue/test-utils'
+import type { DOMWrapper, VueWrapper } from '@vue/test-utils'
+
+type WrappedElement = Omit<DOMWrapper<Element>, 'exists'>
+
+const plugin = (wrapper: VueWrapper) => {
+  function getByClass(className: string): WrappedElement {
+    return wrapper.get(`.${className}`)
+  }
+
+  function getById(testId: string): WrappedElement {
+    return wrapper.get(`#${testId}`)
+  }
+
+  function getByQuery(query: string): WrappedElement {
+    return wrapper.get(query)
+  }
```

Comment:
Why did we even make this function?

Reply:
For now it's a replacement for https://github.com/leaderboardsgg/leaderboard-site/blob/80b78ab74411a660fab23589f773816067ae3094/testUtils.ts#L18-L23

It was used in one spot so I just ported it over.
https://github.com/leaderboardsgg/leaderboard-site/blob/80b78ab74411a660fab23589f773816067ae3094/app/components/blocks/cards/SignUpCard/SignUpCard.test.ts#L32-L37

Technically there are more elegant ways to do this but rewriting other people's tests for the sake of rewriting felt unnecessary, unless the underlaying component was actually worked on. 

So I just focused on fixing the ones that were throwing warnings primarily.

Reply:
You can delete it and replace the one instance with a use of `get()` if you want to. If not, feel free to mark this as resolved.